releasenotes/notes/no-useless-DNS-queries-1ed8ff14e0684da1.yaml

```diff
@@ -6,6 +6,12 @@
 #
 # Each section note must be formatted as reStructuredText.
 ---
+upgrade:
+  - |
+    In order to avoid useless DNS queries, the agent now uses FQDN when connecting to datadog intakes.
+    Concretely, it adds a trailing dot at the end of the datadog intake hostnames.
+    While this change should be transparent to most users, it can have an impact when the agent is deployed on intrusive setups where the connections between the agent and the datadog intakes are intercepted by a proxy or a firewall that is doing deep packet inspection or TLS man-in-the-middle.
```

Comment:
```suggestion
While most users may not notice this change, it can affect setups where connections between the agent and Datadog intakes are intercepted for deep pocket inspection or TLS man-in-the-middle by proxies or firewalls.
```